### PR TITLE
feat: Adding python3.11 support.

### PR DESCRIPTION
**Pull request by awais786**
Adding Python3.11.
Dropped python3.8 support.

### CODE REVIEW

**Comment by awais786** [comment]
@feanil Please review this. Coverage is failing.

**Comment by feanil** [comment]
This is in the edx org and specific to edx.org, I can't make any fixes here.

**Comment by MichaelRoytman** [comment]
@awais786

Can this pull request be reviewed and merged? We are having some trouble merging Python requirements upgrades due to incompatibility with Python 3.8, so it would be good to upgrade. Thank you.

**Comment by awais786** [comment]
> @awais786
> 
> Can this pull request be reviewed and merged? We are having some trouble merging Python requirements upgrades due to incompatibility with Python 3.8, so it would be good to upgrade. Thank you.

@MichaelRoytman its green now. You need to drop the python3.8 support.

**Comment by awais786** [comment]
@MichaelRoytman you can review and merge.

File: edx_name_affirmation/__init__.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 Django app housing name affirmation logic.
 """
 
-__version__ = '2.3.7'
+__version__ = '2.4.0'
```
Comment (ChrisChV) [inline edx_name_affirmation/__init__.py:5]: @MichaelRoytman @UsamaSadiq Could you bump this version on pip? Please. I need to update [edx-opaque-keys](https://github.com/edx/edx-name-affirmation/pull/189/files#diff-0940fec0afe21798f617cc4522db676f15769ae28a72992b8d19216ef7b1c3b5R104) in other repos. Thanks!

Comment (MichaelRoytman) [reply edx_name_affirmation/__init__.py:5]: @ChrisChV Thank you for the ping. I just published a new release. It should be available on PyPI! Please let me know if you need anything else.

Comment (ChrisChV) [reply edx_name_affirmation/__init__.py:5]: Thanks!!

**Pull request merged**